frontend/public/extend/devconsole/components/import/BuilderImageCard.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import { Card, CardHeader, CardBody } from '@patternfly/react-core';
+import { OkIcon } from '@patternfly/react-icons';
+import './BuilderImageCard.scss';
+
+const BuilderImageCard = ({ image, selected, recommended, onChange }) => {
+  return (
+    <Card
```

Comment:
```suggestion
    <Card component="button" aria-label={image.title}
```
You'll have to adjust the styles a bit and fix the removal of the focus outline that console removes in `_base.scss` for whatever reason. But if you do this then you can tab through the buttons instead of just using the mouse.

Reply:
Done.

Reply:
To fix the form submit, you also need to set `type="button"`